.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    env:
+      WHITEPAPER_FORMATS_DIR: whitepaper/formats
+      WHITEPAPER_FILE_NAME_ROOT: gridcoin-whitepaper-draft
+      WHITEPAPER_ARTIFACT_NAME: gridcoin-whitepaper
+    steps:
+    - uses: actions/checkout@v2
+    - name: Update package manager and install o/s level dependencies
+      run: sudo apt update && sudo apt install nodejs npm
+    - name: Install secondary dependencies; '--unsafe-perm' is necessary due to https://github.com/BlueHatbRit/mdpdf/issues/69
+      run: sudo npm install --unsafe-perm mdpdf -g
+    - name: Checkout the project source code
+      run: git checkout
```

Comment:
```suggestion
```
This should be redundant in the CI.

Reply:
So there's something else that goes this already?

Reply:
The `actions/checkout` action already fetches and checks out the repo. I am actually not sure what a `git checkout` without branch does.